cookiecutter.json

```diff
@@ -0,0 +1,16 @@
+{
+    "full_name": "Central AI",
+    "email": "Central.ai@analog.com",
+    "namespace": "",
+    "module_name": "aiproject_python",
+    "python_version": "3.11",
+    "project_folder": "{{ cookiecutter.namespace + '-' + cookiecutter.module_name if cookiecutter.namespace else cookiecutter.module_name }}",
+    "short_description": "Python project of the Central AI team.",
+    "version": "0.1.0",
+    "use_jupyterlab": "n",
+    "use_flake8": "n",
```

Comment:
Can we remove the following redundant options?

- `use_flake8`
- `use_isort`
- `command_line_interface`

Reply:
Removed flake8 and isort. I do think we should keep CLI as an option, if there is a project which runs all their source code through a jupyter notebook for example. I am open to either option but prefer to keep it as an option.